packages/nimble-components/src/icon-base/tests/icons.stories.ts

```diff
@@ -1,6 +1,8 @@
 import type { Meta, StoryObj } from '@storybook/html';
 import { html, repeat } from '@microsoft/fast-element';
 import { DesignSystem } from '@microsoft/fast-foundation';
+// Workaround to force Chromatic TurboSnap to re-run this test when icons are changed
+import '../icon-metadata';
 import * as nimbleIconComponentsMap from '../../icons/all-icons';
```

Comment:
I moved to the approach of specifying `externals` in the GitHub actions config. I was a bit worried whether the relative paths are correct and it's hard to tell if it worked since Chromatic doesn't appear to log anything about that configuration. But in the "Publish to Chromatic" step of [the associated build](https://github.com/ni/nimble/actions/runs/5294124534/jobs/9583114055?pr=1301) it did log that all the icons had changed which dirtied the icon story. I didn't see that in other builds so I'm hopeful that means the change worked.

```
Found affected story files:
  packages/nimble-components/src/icon-base/tests/icons.stories.ts [./src/icon-base/tests/icons.stories.ts]
  packages/nimble-components/src/icons/all-icons.ts [./src/icon-base/tests/icons.stories.ts]
  packages/nimble-components/src/icons/arrow-down-right-and-arrow-up-left.ts [./src/icon-base/tests/icons.stories.ts]
...
```

I'll plan to submit as-is and keep an eye on subsequent builds to make sure I didn't break anything.